docs/troubleshooting/host.md

```diff
@@ -0,0 +1,141 @@
+
+You can check it from the [Embedded Longhorn UI](./harvester.md#access-embedded-rancher-and-longhorn-dashboards).
+
+![attached-volume.png](/img/v1.3/troubleshooting/attached-volume.png)
+
+The manually attached object is attached to a node name instead of the pod name.
+
+You can also use the CLI to retrieve the details of the CRD object `VolumeAttachment`.
+
+Example of a volume that was attached using the Longhorn UI:
+
+```
+- apiVersion: longhorn.io/v1beta2
+  kind: VolumeAttachment
+...
+  spec:
+    attachmentTickets:
+      longhorn-ui:
+        id: longhorn-ui
+        nodeID: node-name
+...
+    volume: pvc-9b35136c-f59e-414b-aa55-b84b9b21ff89
+```
+
+Example of a volume that was attached using the Longhorn CSI driver:
+
+```
+- apiVersion: longhorn.io/v1beta2
+  kind: VolumeAttachment
+  spec:
+    attachmentTickets:
+      csi-b5097155cddde50b4683b0e659923e379cbfc3873b5b2ee776deb3874102e9bf:
+        id: csi-b5097155cddde50b4683b0e659923e379cbfc3873b5b2ee776deb3874102e9bf
+        nodeID: node-name
+...
+    volume: pvc-3c6403cd-f1cd-4b84-9b46-162f746b9667
+```
+
+:::note
+
+Manually attaching a volume to the node is not recommended.
+
+Harvester automatically attaches/detaches volumes based on operations like creating or migrating VM.
+
+:::
+
+#### Workaround 1: Set `Detach Manually Attached Volumes When Cordoned` to `True`
+
+The Longhorn setting [Detach Manually Attached Volumes When Cordoned](https://longhorn.io/docs/1.6.0/references/settings/#detach-manually-attached-volumes-when-cordoned) blocks node draining when there are volumes manually attached to the node.
+
+The default value of this setting depends on the embedded Longhorn version:
+
+| Harvester version | Embedded Longhorn version | Default value |
+| --- | --- | --- |
+| v1.3.1 | v1.6.0 | `true` |
+| v1.4.0 | v1.7.0 | `false` |
+
+Set this option to `true` from the [embedded Longhorn UI](./harvester.md#access-embedded-rancher-and-longhorn-dashboards).
+
+#### Workaround 2: Manually Detach the Volume
+
+Detach the volume using the [embedded Longhorn UI](./harvester.md#access-embedded-rancher-and-longhorn-dashboards).
+
+![detached-volume.png](/img/v1.3/troubleshooting/detached-volume.png)
+
+Once the volume is detached, you can successfully enable Maintenance Mode on the node.
+
+![node-enter-maintenance-mode.png](/img/v1.3/troubleshooting/node-enter-maintenance-mode.png)
+
+### Single-Replica Volumes
+
+Harvester allows you to create custom StorageClasses that describe how Longhorn must provision volumes. If necessary, you can create a StorageClass with the [Number of Replicas](../advanced/storageclass.md#number-of-replicas) parameter set to `1`.
+
+When a volume is created using such a StorageClass and is attached to a node using the CSI driver or other methods, the lone replica stays on that node even after the volume is detached.
```

Comment:
Use inline-code as previously already done to keep consistency across the document.
```suggestion
When a volume is created using such a `StorageClass` and is attached to a node using the CSI driver or other methods, the lone replica stays on that node even after the volume is detached.
```